test/Garnet.test.cluster/RedirectTests/ClusterSlotVerificationTests.cs

```diff
@@ -421,7 +421,7 @@ void SERedisTRYAGAINTest(BaseCommand command)
                     ResetSlot();
                     try
                     {
-                        _ = context.clusterTestUtils.GetServer(requestNodeIndex).Execute("DEL", command.GetSingleSlotKeys.ToArray(), CommandFlags.NoRedirect);
+                        _ = context.clusterTestUtils.GetServer(requestNodeIndex).Execute("DEL", command.GetSingleSlotKeys, CommandFlags.NoRedirect);
```

Comment:
The .ToArray() cannot just be removed; it must be replaced by [.. command.GetSingleSlotKeys]

Reply:
Or alternatively (might be better for performance) you can do explicit cast to ensure that the correct overload is called (i.e.
RedisResult Execute(string command, ICollection<object> args, CommandFlags flags = CommandFlags.None);)

```suggestion
                        _ = context.clusterTestUtils.GetServer(requestNodeIndex).Execute("DEL", (ICollection<object>)command.GetSingleSlotKeys, CommandFlags.NoRedirect);
```

Thanks @TedHartMS for noticing!